### ISSUE TITLE
Pytorch model with a 'sigmoid' op cannot do channel pruning.

### ISSUE THREAD

**Issue by hasuoshenyun** [quic/aimet#964]
I trained a model and it contains 'sigmoid' op, it cannot be compressed with chnnel_prunning.

**Comment by quic-ssiddego** [comment]
@hasuoshenyun is there a specific error you observe?

**Comment by Dennis-Johnson** [comment]
I observed a ```NotImplemented()``` exception raised from AIMET's Winnowing module while using ```sigmoid```, ```max_pool2d```, and ```relu``` from the ```torch.nn.functional``` API. I replaced these three with their ```torch.nn``` alternatives in my custom PyTorch model and was then able to use the channel_pruning function.
